src/Microsoft.OData.Core/ODataMetadataJsonOutputContext.cs

```diff
@@ -30,7 +30,7 @@ internal sealed class ODataMetadataJsonOutputContext : ODataOutputContext
         private Utf8JsonWriter jsonWriter;
 
         /// <summary>The asynchronous output stream if we're writing asynchronously.</summary>
-        private AsyncBufferedStream asynchronousOutputStream;
+        private BufferedStream asynchronousOutputStream;

```

Comment:
@henning-krause How come in this file there are no separate blocks for NETSTANDARD1_1 and other target frameworks?

Reply:
@gathogojr this entire file is already inside an `#if NETSTANDARD2_0` block. [See line 7](https://github.com/OData/odata.net/pull/2222/files/0744795483c6c1cdc0dd004937856baee44cb180#diff-75ae7b4982563574b522a0a9b29849fb3b47c2c5f6269b608fa5f965b285303aR7)